presc/evaluations/spatial_distribution.py

```diff
@@ -0,0 +1,343 @@
+from collections import defaultdict
+
+from pandas import DataFrame, Series, MultiIndex
+from sklearn.metrics import pairwise_distances
+from sklearn.preprocessing import StandardScaler, OrdinalEncoder
+import seaborn as sns
+from confuse import ConfigError
+
+from presc.evaluations.utils import is_discrete
+from presc.utils import include_exclude_list
+from presc.configuration import PrescConfig
+from presc import global_config
+
+
+def _pairwise_dist(df1, df2, metrics):
+    """Compute pairwise distances between the rows of two DFs with common columns.
+
```

Comment:
Add Parameters here
       ------------

Reply:
Sorry - missed that it was a private function so it won't be included in the docs anyway.

Reply:
Still, nice to just add the header in as everything else is already formatted.